core/jazz_ui/src/app/create-service/create-service-aws-events/create-service-aws-events.component.ts

```diff
@@ -0,0 +1,59 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {environment} from "../../../environments/environment";
+import {EventExpression} from "../create-service/service-form-data";
+
+@Component({
+  selector: 'create-service-aws-events',
+  templateUrl: './create-service-aws-events.component.html',
+  styleUrls: ['./create-service-aws-events.component.scss']
+})
+export class CreateServiceAwsEventsComponent implements OnInit {
+  @Input() type = 'awsEventsNone';
+  @Output() setAWSEvents = new EventEmitter();
+
+
+  public sqsStreamString: string = "arn:aws:sqs:us-west-2:" + environment.aws.account_number + ":stream/";
```

Comment:
why hardcoding the regions?

Reply:
I was using an old copy of the files as reference. Will update.